lib/ae_mdw_web/controllers/tx_controller.ex

```diff
@@ -58,7 +61,32 @@ defmodule AeMdwWeb.TxController do
     end
   end
 
-  def read_tx_hash(tx_hash) do
+  #
+  # Private functions
+  #
+  defp add_spendtx_details(data_txs, %{"account" => account}) do
+    Enum.map(data_txs, fn block ->
```

Comment:
We're iterating transactions (`data_txs`) but we're naming each of this transactions `block`. So, are they transactions or blocks?

Reply:
they are blocks with transactions with. I will call bock_txs.

Reply:
and renamed data_txs to response_data